app/assets/stylesheets/custom/govuk-table.scss

```diff
@@ -19,4 +19,10 @@
 
 .govuk-table .govuk-table__foot .govuk-table__row .govuk-table__footer:last-child {
   padding-right: 0;
-}
\ No newline at end of file
+}
+
+.govuk-table .govuk-table__cell {
+  max-width: 300px;
+  word-wrap: break-word;
+  white-space: break-spaces;
+}
```

Comment:
So this broke other pages...

<img width="998" alt="Screenshot 2024-08-16 at 11 34 17" src="https://github.com/user-attachments/assets/0eea5488-ccb5-40af-8c3d-7f4bacbcf323">

Reply:
Was afraid of that, the one table I didn't check 🤣